package.json

```diff
@@ -30,6 +30,7 @@
     "helpers:test": "turbo run test --filter=@asyncapi/generator-helpers",
     "helpers:lint": "turbo run lint --filter=@asyncapi/generator-helpers",
     "templates:test": "turbo run test --filter=@asyncapi/template*",
+    "packages:test": "turbo run test --filter=./packages/** --only",
```

Comment:
this was added to have one command in monorepo to run all tests but only for packaged directory